lib/pbio/drv/usb/usb_nxt.c

```diff
@@ -27,6 +27,12 @@
 #include "nxos/drivers/aic.h"
 #include "nxos/util.h"
 
+#include <lego/usb.h>
+#include "pbdrvconfig.h"
```

Comment:
```suggestion
#include <pbdrv/config.h>
```

We never include `pbdrvconfig.h` directly.